tex_inspection/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "tex-inspection"
+version = "0.1.0"
+description = "Inspecting TeX in arXiv submissions"
+authors = ["arxiv.org"]
+license = "BSD-3 License"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+"ruamel.yaml" = "^0.18"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^8.1.1"
+mypy = "*"
+mypy-extensions = "*"
+pydantic = "^2.6.4"
```

Comment:
This will conflict with base in other projects. Please use pydantic = "1.10.*" if possible

Reply:
NP.